src/ontology/mondo-edit.obo

```diff
@@ -464918,6 +464920,2857 @@ is_a: MONDO:0005550 {source="https://orcid.org/0000-0002-5002-8648"} ! infectiou
+intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0018149 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog GM1 gangliosidosis
+
+[Term]
+id: MONDO:1010169
+name: glycogen storage disease II, non-human animal
+def: "Glycogen storage disease II that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:000419 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000419"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0009290 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog glycogen storage disease II
+
+[Term]
+id: MONDO:1010170
+name: Krabbe disease, non-human animal
+def: "Krabbe disease that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:000578 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000578"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0009499 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog Krabbe disease
+
+[Term]
+id: MONDO:1010171
+name: lysosomal storage disease, non-human animal
+def: "Lysosomal storage disease that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:000616 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000616"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0002561 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog lysosomal storage disease
+
+[Term]
+id: MONDO:1010172
+name: alpha-mannosidosis, non-human animal
+def: "Alpha-mannosidosis that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:000625 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000625"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:000625"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0009561 {source="OMIA:000625"} ! cross-species analog alpha-mannosidosis
+
+[Term]
+id: MONDO:1010173
+name: mucopolysaccharidosis IIID, non-human animal
+def: "Mucopolysaccharidosis IIID that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:000665 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000665"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:000665"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0009658 {source="OMIA:000665"} ! cross-species analog mucopolysaccharidosis type 3D
+
+[Term]
+id: MONDO:1010174
+name: mucopolysaccharidosis VI, non-human animal
```

Comment:
In human, "disease ~ type x" is based on the gene affected. Should we have only the species-specific term(s) instead of this generic term based on "general species unspecific" gene? 
Note that these species-specific diseases (based on species-specific genes) could be analogous to the human type x disease, but it is difficult to create an axiom referring to the general "species-not- specific" gene.